src/Routers/UsersRouter.js

```diff
@@ -162,13 +182,20 @@ export class UsersRouter extends ClassesRouter {
             delete user.authData;
           }
         }
+
+        // Remove hidden properties.
+        UsersRouter.removeHiddenProperties(user);
```

Comment:
this should be moved to the `_handleValidatePassword` method as also shared with the other call.